.gitignore

```diff
@@ -5,6 +5,10 @@ __pycache__/
 
 # C extensions
 *.so
+*.dylib
```

Comment:
I'm not sure if there are any other files with these extensions in our repo. Should be consider explicitly excluding these files from `src\debugpy\_vendored\pydevd\pydevd_attach_to_process` ?

Reply:
There isn't any but I think we shouldn't ever be checking in dlls/exes. Too opaque.